service/gateway/router.go

```diff
@@ -144,6 +156,16 @@ func (g *Gateway) registerHandler(r *mux.Router) {
 		Path("/{object:.+}").
 		Queries(model.UploadProgressQuery, "").
 		HandlerFunc(g.queryUploadProgressHandler)
+	pathBucketRouter.NewRoute().
+		Name(getBucketAndPaymentRouterName).
+		Methods(http.MethodGet).
+		Path("/bucket/payment").
+		HandlerFunc(g.getBucketWithPaymentHandler)
```

Comment:
what happens if bucketName=`bucket` and objectName=`payment`?

Reply:
Fixed, using query param instead.